README.md

```diff
@@ -133,8 +133,7 @@ end
 ```ruby
 #!/usr/bin/env ruby
 
-$LOAD_PATH.unshift(File.expand_path('../../lib',__FILE__))
-require 'foo/cli/my_cmd'
+require_relative 'lib/foo/cli/my_cmd'
```

Comment:
This probably won't work because then other `require`s that expect the `lib/` directory to be added to `$LOAD_PATH` would fail. By explicitly adding `lib/` to `$LOAD_PATH` it allows executing the `./bin/` executable directory without bundler.

Reply:
I'm not sure what you mean. For example for https://github.com/postmodern/command_kit.rb/blob/main/examples/command.rb, `$LOAD_PATH.unshift(File.expand_path('../../lib',__FILE__))` is unneeded. There is no need for `require_relative` either. `require 'command_kit/command'` is enough. If the gem `command_kit` is installed, then it will work. And if it is not installed but you want to load from the current directory (for example someone that git cloned), you can just load if with `irb -Ilib -rcommand_kit` or `ruby -Ilib -rcommand_kit examples/command.rb`.